server/server/src/schema.rs

```diff
@@ -45,7 +45,8 @@ pub mod v2 {
             file_children: <Uuid, HashSet<Uuid>>,
             sizes: <Uuid, u64>,
             google_play_ids: <String, Owner>,
-            stripe_ids: <String, Owner>
+            stripe_ids: <String, Owner>,
+            app_store_ids: <String, Owner>
```

Comment:
This needs to end up in a v3 with a migration, see the rest of this file. Also see how this is gonna interact with the proposed schema changes [here](https://github.com/lockbook/lockbook/issues/1296#issuecomment-1368230780)

cc: @ad-tra 

I'm down to not do it in this PR and let @ad-tra just do it in his PR